### PR TITLE
Remove references to the editor

### PR DESCRIPTION
**Pull request by jmhooper**
This commit removes references to the Federalist editor since the editor has been removed.

Ref 18f/federalist#770

### CODE REVIEW

File: pages/custom-templates.md [hunk 1]
```diff
@@ -23,66 +21,6 @@ Federalist provides features beyond just generating Jekyll sites. The steps belo
 
 Federalist adds a `site.branch` attribute to your global site object with the value of the current branch name. You can access this value in your templates and content and use it to style builds based on the working branch.
 
-Federalist also adds information about the git commit at which the site was built. Access this information through `site.commit`. The commit object includes `site.commit.commit` (the SHA), `site.commit.author`, `site.commit.date`, and `site.commit.message`.
```
Comment (wslack) [inline pages/custom-templates.md]: should this part be deleted?

Comment (jmhooper) [inline pages/custom-templates.md]: Yes, but maybe in a different PR. This is something we removed b/c it was buggy.

Comment (wslack) [inline pages/custom-templates.md]: so do you want to alter this branch or go ahead and merge?

Comment (jmhooper) [inline pages/custom-templates.md]: I'll move it to it's own PR

Comment (jmhooper) [inline pages/custom-templates.md]: I moved this change to #59

**Pull request merged**